docs/cluster_install.md

```diff
@@ -0,0 +1,65 @@
+# Setup UCX from Databricks web terminal
+For cases when your desktop or laptop will not support the `databricks labs install ucx` technical requirements such as Python 3.10, administrative access, python package access or network access but your Databricks cluster will have the internet access requirements same as described in the main README.md document.
+- Start an *Unassigned* interactive cluster 
+- Create a new notebook to attach to the cluster
+- Open the web terminal, expand to full page for better readability
+
+Install and setup needed UCX dependencies:
+
+```shell
+apt install python3.10-venv
+unset DATABRICKS_RUNTIME_VERSION
+```
+
+Download and install Databricks cli:
+
+```shell
+curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/main/install.sh | sh
+```
+
+Set PATH variable to point to new install:
+
+```shell
+export PATH=/usr/local/bin:$PATH
+```
+
+Verify Databricks cli:
+
+```shell
+which databricks
+databricks --version
+```
+
+Configure Databricks cli to authenticate to correct workspace, add host name and PAT token:
+```shell
+databricks configure
```

Comment:
This is security hole, as this configuration is accessible to all users on the same cluster. No way we're recommending this

Reply:
cc @hamza-db

Reply:
We can specify a single user cluster and denying access to the cluster.

Reply:
@nfx any alternatives you'd suggest?

Reply:
Ok, Environmental variable is session based, not available to other shells.

Reply:
users by mistake run it on a shared cluster and then it's root - everyone, they leak admin credentials for all users. no.